packages/graphiql-react/src/style/root.css

```diff
@@ -73,39 +59,25 @@ reach-portal {
   body:not(.graphiql-light) .CodeMirror-info,
   body:not(.graphiql-light) .CodeMirror-lint-tooltip,
   body:not(.graphiql-light) reach-portal {
-    --color-pink: #ff5895;
-    --color-purple: #918cff;
-    --color-blue: #6fbcff;
-    --color-orche: #ffcc99;
-    --color-neutral-100: #b7c2d7;
-    --color-neutral-60: #5a687f;
-    --color-neutral-40: #465877;
-    --color-neutral-15: #2d384c;
-    --color-neutral-10: #26303f;
-    --color-neutral-7: #293242;
-    --color-neutral-0: #222b39;
-
-    --color-pink-dark: #ff357f;
+    --color-pink: 338, 100%, 67%;
+    --color-purple: 243, 100%, 77%;
+    --color-blue: 208, 100%, 72%;
+    --color-orche: 30, 100%, 80%;
+    --color-neutral: 219, 29%, 78%;
+    --color-base: 219, 29%, 18%;
   }
 }
 
 body.graphiql-dark .graphiql-container,
 body.graphiql-dark .CodeMirror-info,
 body.graphiql-dark .CodeMirror-lint-tooltip,
 body.graphiql-dark reach-portal {
-  --color-pink: #ff5895;
-  --color-purple: #918cff;
-  --color-blue: #6fbcff;
-  --color-orche: #ffcc99;
-  --color-neutral-100: #b7c2d7;
-  --color-neutral-60: #5a687f;
-  --color-neutral-40: #465877;
-  --color-neutral-15: #2d384c;
-  --color-neutral-10: #26303f;
-  --color-neutral-7: #293242;
-  --color-neutral-0: #222b39;
-
-  --color-pink-dark: #ff357f;
+  --color-pink: 338, 100%, 67%;
+  --color-purple: 243, 100%, 77%;
+  --color-blue: 208, 100%, 72%;
+  --color-orche: 30, 100%, 80%;
+  --color-neutral: 219, 29%, 78%;
```

Comment:
This is where we may run into extensibility issues in the future. I did some testing auto-generating dark mode values was less reliable than it was in light mode. It's a well-understood dark theme problem, and we can decide how to deal with it down the road.